packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.tsx

```diff
@@ -112,15 +105,6 @@ const WalletDropdown: React.FC<TProps> = ({
                     value={value}
                     {...getInputProps()}
                 />
-                <label
```

Comment:
Figure we dont need this since label is already in WalletTextField

Reply:
And other bug fixes in dropdown is handled here https://github.com/binary-com/deriv-app/pull/11549